src/generateTypeFiles.js

```diff
@@ -76,7 +76,7 @@ export const generateTypeFileContents = (
             (options.regenerateCommand
                 ? `// To regenerate, run '${options.regenerateCommand}'.\n`
                 : '') +
-            code.replace(/\s+$/g, '');
+            code.replace(/\s+$/gm, '');
```

Comment:
This is a better use for a regex now that it's trimming the ends of all lines.  No action req'd.